js/modules/timeline.src.js

```diff
@@ -73,7 +90,25 @@ seriesType('timeline', 'line', {
             '{point.label}',
         borderWidth: 1,
         borderColor: 'gray',
+        /**
+         * A distance between the data label and the point's edge.
+         * Negative numbers puts the data label on top of the point.
+         *
+         * @type {Number}
+         * @default 100
+         */
         distance: 100,
```

Comment:
Missed information if distance is in pixels or different measure unit.

Example -> https://api.highcharts.com/highcharts/annotations.labels.distance

Reply:
Information about measure unit added.